### PR TITLE
Small typography changes

### PR DESCRIPTION
**Pull request by dawsoneliasen**
- Reduce navbar font size
- Reduce footer font size
- Change "800,000" to "250,000,000"
- Change body font to Roboto
- Change subheader on hero photo to Roboto
- Change from "have not achieved" to "have not yet achieved." 
- Change "The Oppia Foundation is working to change that." from green to white
- Remove "we are building a global classroom"

### CODE REVIEW

File: static/pages/home/home.html [hunk 1]
```diff
@@ -100,7 +100,7 @@ <h3 style="margin-top: 0;">Impacting students around the world</h3>
     <md-card-content layout="row" layout-align="center">
       <md-content layout="column" class="oppia-foundation-white-text oppia-foundation-green-background home-page-history-text-box">
         <p class="oppia-foundation-center-align-text">
-          Since launching Oppia.org in 2015, over 800,000 learners in more than 20 countries have used our platform to explore, grow, and learn.
+          Since launching Oppia.org in 2015, over one million learners in more than 20 countries have used our platform to explore, grow, and learn.
```
Comment (hoangviet1993) [inline static/pages/home/home.html]: Hmmm, I think I used the numeric count (`1,000,000`) over in Partnerships page. Do you want to apply this changes to `partnerships.html` too?

Comment (dawsoneliasen) [reply static/pages/home/home.html]: Sure thing, easy do.

**Comment by dawsoneliasen** [comment]
screen shot 2018-09-06 at 12 08 14 pm
screen shot 2018-09-06 at 12 08 32 pm
screen shot 2018-09-06 at 12 08 48 pm
screen shot 2018-09-06 at 12 08 59 pm
screen shot 2018-09-06 at 12 09 14 pm
screen shot 2018-09-06 at 12 09 29 pm

File: static/pages/home/home.html (hunk 1 above)
Comment (dawsoneliasen) [reply static/pages/home/home.html]: Oh, do you want them both to say "one million" or "1,000,000"?

Comment (hoangviet1993) [reply static/pages/home/home.html]: I don't really have a strong preference here. I do feel like the numeric version has stronger impact. That is just me though.

File: static/pages/home/home.html [hunk 2]
```diff
@@ -58,6 +58,13 @@ <h3 class="md-headline oppia-foundation-headline">
       </div>
     </div>
   </md-card>
+<<<<<<< HEAD
```
Comment (hoangviet1993) [inline static/pages/home/home.html]: You miss these!

Comment (dawsoneliasen) [reply static/pages/home/home.html]: Argh, sorry about that. Should be good now.

**Comment by hoangviet1993** [comment]
@dawsoneliasen can you do a git pull from upstream? your branch is missing the config file for CircleCI

File: static/pages/home/home.html [hunk 3]
```diff
@@ -340,28 +334,24 @@ <h3 class="md-display-1">
...
     <md-card-content class="home-page-display-size
                             oppia-foundation-no-padding"
-                     layout="column"
-                     layout-align="center center">
+                      layout="column" 
+                      layout-align="center center">
       <md-content class="oppia-foundation-black-text
                          oppia-foundation-white-background
-                         home-page-description-text-box"
+                         home-page-donation-text-box"
```
Comment (hoangviet1993) [inline static/pages/home/home.html]: I think this is a left-over from the git conflict. Can you check if this css class still exists?

File: static/pages/home/home.html [hunk 4]
```diff
@@ -340,28 +334,24 @@ <h3 class="md-display-1">
...
+                      layout="column" 
+                      layout-align="center center">
       <md-content class="oppia-foundation-black-text
                          oppia-foundation-white-background
-                         home-page-description-text-box"
+                         home-page-donation-text-box" 
                   layout="column"
                   layout-align="center center">
-        <center>
-          <h3 class="md-display-1">
```
Comment (hoangviet1993) [inline static/pages/home/home.html]: Why did you delete the md-display-1 class?

File: static/pages/home/home.html [hunk 5]
```diff
@@ -45,37 +39,27 @@ <h4 class="md-headline oppia-foundation-headline">
...
-          More than 250 million children worldwide have not achieved basic 
-          literacy and numeracy.
-        </p>
-        <p class="oppia-foundation-green-text">
-          The Oppia Foundation is working to change that.
-        </p>
+      <md-content layout="column" 
+                  class="oppia-foundation-white-background 
+                        oppia-foundation-center-align-text">
+        <p>More than 1,000,000 children worldwide have not yet achieved basic literacy and numeracy.</p>
```
Comment (hoangviet1993) [inline static/pages/home/home.html]: @seanlip Is this correct? 250m down to just 1m?

File: static/pages/home/home.html [hunk 6]
```diff
@@ -340,28 +324,23 @@ <h3 class="md-display-1">
            layout-fill>
     <md-card-content class="home-page-display-size
                             oppia-foundation-no-padding"
-                     layout="column"
-                     layout-align="center center">
+                      layout="column" 
+                      layout-align="center center">
       <md-content class="oppia-foundation-black-text
-                         oppia-foundation-white-background
-                         home-page-description-text-box"
```
Comment (hoangviet1993) [inline static/pages/home/home.html]: Class deleted by accident?

File: static/pages/home/home.html (hunk 5 above)
Comment (dawsoneliasen) [reply static/pages/home/home.html]: Ah okay, my bad.

File: static/pages/home/home.html (hunk 6 above)
Comment (dawsoneliasen) [reply static/pages/home/home.html]: I guess I misunderstood your feedback earlier: https://github.com/oppia/foundation-website/pull/41#discussion_r216429304 

I thought that the class needed to be deleted.

Comment (hoangviet1993) [reply static/pages/home/home.html]: This? https://github.com/oppia/foundation-website/pull/41#discussion_r216429304
Hm, can you check if there is any visual changes with and without that css class?

Comment (dawsoneliasen) [reply static/pages/home/home.html]: Yes, that's what I'm referring to. There doesn't appear to be any difference.

Comment (hoangviet1993) [reply static/pages/home/home.html]: Are you sure?
screen shot 2018-09-11 at 2 00 33 pm

vs

screen shot 2018-09-11 at 2 00 17 pm

**Comment by dawsoneliasen** [comment]
@hoangviet1993 PTAL

File: static/pages/home/home.html [hunk 7]
```diff
@@ -340,28 +324,24 @@ <h3 class="md-display-1">
            layout-fill>
     <md-card-content class="home-page-display-size
                             oppia-foundation-no-padding"
-                     layout="column"
-                     layout-align="center center">
+                      layout="column" 
+                      layout-align="center center">
```
Comment (hoangviet1993) [inline static/pages/home/home.html]: I don't think you need to add spaces here?

File: static/pages/home/home.html [hunk 8]
```diff
@@ -340,28 +324,24 @@ <h3 class="md-display-1">
...
       <md-content class="oppia-foundation-black-text
                          oppia-foundation-white-background
-                         home-page-description-text-box"
+                         home-page-description-text-box"
```
Comment (hoangviet1993) [inline static/pages/home/home.html]: Unnecessary space added here.

**Comment by hoangviet1993** [comment]
This is optional but I find the social media icon disproportionate (too big) in comparison to the smaller footer link.
It is just my opinion though and we can come back to fix it later.
screen shot 2018-09-11 at 11 45 29 pm

**Comment by dawsoneliasen** [comment]
@hoangviet1993 PTAL; whitespace is fixed. I agree with your sentiment regarding the icons, but let's save it for later.

**Comment by hoangviet1993** [comment]
Thanks! @dawsoneliasen
